Packs/FeedMitreAttackv2/Integrations/FeedMitreAttackv2/FeedMitreAttackv2.py

```diff
@@ -446,7 +510,7 @@ def get_indicators_command(client, args):
             'Type': entryTypes['note'],
             'Contents': indicators,
             'ContentsFormat': formats['json'],
-            'HumanReadable': tableToMarkdown('MITRE ATT&CK v2 Indicators:', indicators, ['value', 'score', 'type']),
+            'HumanReadable': tableToMarkdown('MITRE ATT&CK v2 Indicators:', indicators[0], ['value', 'score', 'type']),
```

Comment:
Why just the first item?

Reply:
the dictionary comes back in index 0
![image](https://github.com/user-attachments/assets/4d34f8fc-5169-4112-819b-110264529e49)